gcloud_utils/ml_engine.py

```diff
@@ -116,6 +117,40 @@ def create_new_model(self, name, description='Ml model'):
             parent=self.parent, body=request_dict)
         return request
 
+    def export_model(self, clf, model_path="model.pkl"):
+        """
+        Export a classifier/pipeline to model path.
+        Frameworks supported : XGBoost booster, Scikit-learn estimator and pipelines.
+        """
+        try:
+            with open(model_path, 'wb') as model_file:
+                pickle.dump(clf, model_file)
+        except Exception as error:
+            logging.error("Failed to export model to {}.".format(model_path))
+            raise error
+
+    def predict_json(self, project, model, instances, version=None):
+        """Send json data to a deployed model for prediction.
+          Returns:
```

Comment:
Please, remove the returns from the docstring too.
We have to improve this docstring later.